parachain/pallets/xcm-support/Cargo.toml

```
@@ -0,0 +1,49 @@
+[package]
+name = "snowbridge-xcm-support"
+description = "Snowbridge XCM Support Pallet"
+version = "0.1.1"
+edition = "2021"
+authors = [ "Snowfork <contact@snowfork.com>" ]
+repository = "https://github.com/Snowfork/snowbridge"
+
+[package.metadata.docs.rs]
+targets = [ "x86_64-unknown-linux-gnu" ]
+
+[dependencies]
+codec = { version = "3.0.0", package = "parity-scale-codec", features = [ "derive" ], default-features = false }
+scale-info = { version = "2.0.1", default-features = false, features = [ "derive" ] }
+
+frame-support = { git = "https://github.com/paritytech/substrate", default-features = false, branch = "polkadot-v0.9.19" }
+frame-system = { git = "https://github.com/paritytech/substrate", default-features = false, branch = "polkadot-v0.9.19" }
+sp-core = { git = "https://github.com/paritytech/substrate", default-features = false, branch = "polkadot-v0.9.19" }
+sp-runtime = { git = "https://github.com/paritytech/substrate.git", branch = "polkadot-v0.9.19", default-features = false }
+sp-std = { git = "https://github.com/paritytech/substrate.git", branch = "polkadot-v0.9.19", default-features = false }
+pallet-xcm = { git = "https://github.com/paritytech/polkadot.git", branch = "release-v0.9.19", default-features = false }
+
+xcm = { git = "https://github.com/paritytech/polkadot.git", branch = "release-v0.9.19", default-features = false }
+xcm-executor = { git = "https://github.com/paritytech/polkadot.git", branch = "release-v0.9.19", default-features = false }
+xcm-builder = { git = "https://github.com/paritytech/polkadot.git", branch = "release-v0.9.19", default-features = false }
+
+snowbridge-xcm-support-primitives = { path = "../../primitives/xcm-support", default-features = false }
+
+[dev-dependencies]
+serde = { version = "1.0.136" }
+sp-core = { git = "https://github.com/paritytech/substrate", default-features = false, branch = "polkadot-v0.9.19" }
+sp-runtime = { git = "https://github.com/paritytech/substrate", default-features = false, branch = "polkadot-v0.9.19" }
+sp-io = { git = "https://github.com/paritytech/substrate", default-features = false, branch = "polkadot-v0.9.19" }
+
+[features]
+default = [ "std" ]
+std = [
+	"codec/std",
+	"scale-info/std",
+	"frame-support/std",
+	"frame-system/std",
```

I think sp-core is missing here:

```suggestion
	"frame-system/std",
	"sp-core/std",
```